reference-implementation/test/readable-stream-templated.js

```diff
@@ -20,20 +19,6 @@ templatedRSClosed('ReadableStream (closed via cancel)',
 
 const theError = new Error('boo!');
 
-templatedRSErrored('ReadableStream (errored via call in start)',
```

Comment:
you've chose not to port the instance for this line? I think it's ok if the basic functionality of controller.error() is tested somewhere else, but just to make sure you're aware of the removal.